README.rst

```diff
@@ -189,7 +189,7 @@ the ``--age`` option or the **SOPS_AGE_RECIPIENTS** environment variable:
 
 .. code:: bash
 
-   $ sops --encrypt --age age1yt3tfqlfrwdwx0z0ynwplcr6qxcxfaqycuprpmy89nr83ltx74tqdpszlw test.yaml > test.enc.yaml
+   $ sops --encrypt --age age1yt3tfqlfrwdwx0z0ynwplcr6qxcxfaqycuprpmy89nr83ltx74tqdpszlw -e test.yaml > test.enc.yaml
```

Comment:
`-e` is short for `--encrypt`, which is already present at the beginning of the line.

Reply:
It is missing in Master branch's README.